src/js/game/systems/item_processor.js

```diff
@@ -31,7 +32,7 @@ export class ItemProcessorSystem extends GameSystemWithFilter {
 
             // Check if we have any finished items we can eject
             if (
-                processorComp.secondsUntilEject === 0 && // it was processed in time
+                quantizeFloat(processorComp.secondsUntilEject) == 0 && // it was processed in time
```

Comment:
Why remove the strict check? Under the hood `quantizeFloat` performs a `Math.round()` operation which has a return type of `number`. This matches the data type of `processorComp.secondsUntilEject` so the struck check should still work.

Reply:
Same question for your changes to `underground_belt.js` both changes arr to properties with a `number` data type.